src/cnv/ugbio_cnv/cnmops_utils.py

```diff
@@ -1,118 +1,16 @@
-        "the tool called this CNV. can be combination of: cn.mops, cnvpytor, gridss",
-        "INFO",
-    ),
-    "JUMP_ALIGNMENTS": ("JUMP_ALIGNMENTS", 1, "Float", "Number of jump alignments supporting this CNV", "INFO"),
-    "CNMOPS_SAMPLE_MEAN": (
-        "CNMOPS_SAMPLE_MEAN",
-        1,
-        "Float",
-        "Mean coverage in the CNV region for the sample (cn.mops)",
-        "INFO",
-    ),
-    "CNMOPS_SAMPLE_STDEV": (
-        "CNMOPS_SAMPLE_STDEV",
-        1,
-        "Float",
-        "Standard deviation of coverage in the CNV region for the sample (cn.mops)",
-        "INFO",
-    ),
-    "CNMOPS_COHORT_MEAN": (
-        "CNMOPS_COHORT_MEAN",
-        1,
-        "Float",
-        "Mean coverage in the CNV region across the cohort (cn.mops)",
-        "INFO",
-    ),
-    "CNMOPS_COHORT_STDEV": (
-        "CNMOPS_COHORT_STDEV",
-        1,
-        "Float",
-        "Standard deviation of coverage in the CNV region across the cohort (cn.mops)",
-        "INFO",
-    ),
-    "GAP_PERC": ("GAP_PERC", 1, "Float", "Fraction of N bases in the CNV region from reference genome", "INFO"),
-}
-
-# the reason filters require special treatment is that they need to be
-# unique and should be PASS if none present. In the end filter tags are added to info
-# All columns from FILTER_COLUMNS_REGISTRY are aggregated into a single INFO field FILTER_ANNOTATION_NAME
-FILTER_COLUMNS_REGISTRY = ["LCR_label_value"]
-FILTER_ANNOTATION_NAME = "REGION_ANNOTATIONS"
-INFO_TAG_REGISTRY[FILTER_ANNOTATION_NAME] = (
-    "REGION_ANNOTATIONS",
-    ".",
-    "String",
-    "Aggregated region-based annotations for the CNV (e.g., LCR status and other region filters)",
-    "INFO",
+from ugbio_cnv.cnv_consts import (
+    FILTER_COLUMNS_REGISTRY,
+    FILTER_TAG_REGISTRY,
+    INFO_TAG_REGISTRY,
 )
 
-FILTER_TAG_REGISTRY = {
-    "Clusters": ("Clusters", None, None, "Overlaps with locations with frequent clusters of CNV", "FILTER"),
-    "Coverage-Mappability": (
-        "Coverage-Mappability",
-        None,
-        None,
-        "Overlaps with low coverage or low mappability regions",
-        "FILTER",
-    ),
-    "Telomere_Centromere": (
-        "Telomere_Centromere",
-        None,
-        None,
-        "Overlaps with telomere or centromere regions",
-        "FILTER",
-    ),
-    "LEN": (
-        "LEN",
-        None,
-        None,
-        "CNV length is below the minimum length threshold (cn.mops)",
-        "FILTER",
-    ),
-    "UG-CNV-LCR": (
-        "UG-CNV-LCR",
-        None,
-        None,
-        "Overlaps with low-complexity regions as defined by UGBio CNV module",
-        "FILTER",
-    ),
-    "CNMOPS_SHORT_DUPLICATION": (
-        "CNMOPS_SHORT_DUPLICATION",
-        None,
-        None,
-        "Duplication length is shorter than the defined threshold in cn.mops calls.",
-        "FILTER",
-    ),
-}
+warnings.filterwarnings("ignore")
 
 
 def add_vcf_header(sample_name: str, fasta_index_file: str) -> pysam.VariantHeader:
```

Comment:
is this the right place for this function? it is not cnmops specific .

Reply:
I think it is cn.mops specific, in other places we do not create vcf from bed